package.json

```diff
@@ -10,7 +10,7 @@
     "lib/index.es.js"
   ],
   "engines": {
-    "node": ">=4"
+    "node": "4.x || >=6.0.0"
```

Comment:
Since it's quite a hassle to test a warning message in CI, could you try use [n
](https://github.com/tj/n) to verify it produces the warning message correctly (use `npm install <folder>`)

Other than that, this change LGTM